.github/workflows/test.yaml

```diff
@@ -32,6 +32,11 @@ jobs:
         steps:
         - uses: actions/checkout@v2
 
+        - name: Get history and tags for SCM versioning to work
+          run: |
+            git fetch --prune --unshallow
+            git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+
```

Comment:
What is this addition doing?

Reply:
This is needed if we include `ape-solidity` in `setup.py`. Otherwise, `ape` has a weird version in the GH Action. Anything dependent on `ape`, like plugins, are unable to install when `ape` has a weird, non-alpha version that.